native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -36,14 +36,19 @@ const styles = StyleSheet.create({
   horizontalNav: {
     borderColor: 'transparent',
     borderBottomColor: colorGray100,
+    flexDirection: 'column',
     borderWidth: borderSizeSm,
   },
+  // to allow ScrollView item to be properly positioned in the spaceAround config
+  // the contentContainerStyle needs flexGrow to have the children define the end
+  // width and allow overflow from the view
+  flexGrow: {
+    flexGrow: 1,
+  },
   horizontalNavInner: {
     flexDirection: 'row',
   },
   spaceAround: {
```

Comment:
Can we rename the styles from:
```
horizontalNav: {
},
flexGrow: {
},
horizontalNavInner: {
},
spaceAround: {
}
```

to:

```
nav: {
},
navSpaceAround: {
},
inner: {
},
innerSpaceAround: {
}
```

By prefixing the names it's easier to see what maps where.